Add pointer to Private Network Access update. (#5087)
GoogleChrome/web.dev: src/site/content/en/blog/cors-rfc1918-feedback/index.md

  - cors
---

{% Aside %}

>   An update to this post was published to the [developer.chrome.com
>   blog](https://developer.chrome.com/blog/private-network-access-update).
>   CORS-RFC1918 has also been renamed to Private Network Access for clarity.

Reviewer: ```suggestion
  CORS-RFC1918 has been renamed to Private Network Access for clarity.
  An update to this post is published at [developer.chrome.com
  blog](https://developer.chrome.com/blog/private-network-access-update).
```
